homeassistant/components/alert.py

```diff
@@ -33,7 +33,7 @@
 DEFAULT_SKIP_FIRST = False
 
 ALERT_SCHEMA = vol.Schema({
-    vol.Required(CONF_NAME): cv.string,
+    vol.Optional(CONF_NAME): cv.string,
```

Comment:
What will the name be if it's not given? It looks like that if the name is not given the service will no longer be called.

Reply:
Yeah, I realised that it is unfortunately a bit more work than I hoped for. So it might be a bit over my head.
I'll give it a go later this week when I have an opportunity to test.